### ISSUE TITLE
JITSI-MEET SERVER IN A VPS WITH UBUNTU SERVER NO AUDIO NO VIDEO.

### ISSUE THREAD

**Issue by eduardo606** [jitsi/jitsi-meet#606]
I've installed Jitsi-meet server in a Virtual Private Server (VPS) using the quick install.

echo 'deb http://download.jitsi.org/nightly/deb unstable/' >> /etc/apt/sources.list
wget -qO - https://download.jitsi.org/nightly/deb/unstable/archive.key | apt-key add -
apt-get update
apt-get -y install jitsi-meet

The VPS has a private ip address 192.168.x.x and has a public Ip address 200.x.x.x.
In the server there is a NAT configured between the private address 192.168.x.x and the public address 200.x.x.x
I have local access to the server using a VCLOUD interface,
I open a chromium browser and type the address 192.168.x.x for create a jitsi-meet room,
From a client i use a google chrome browser and type the address 200.x.x.x for join me to the jitsi-meet room.
But i can't see the participant in the server browser, and i can't see the participant name in the contact list.
In the client there isn't audio, there isn't video, the control buttons at the top of the interface of Jitsi -meet dont work.
in the client i can't write the participant name because this is deleted immediately.
in the client i can't see the server name. i can't register the client name, .

In summary the client can't see the server and the server can't see the client.

I know that there is similar issues reported, but i've tried all solutions, and my jitsi-meet server don't work.

you can answer in this forum or in my e-mail tussin0610@gmail.com

I hope you can help me.

thanks.

**Comment by damencho** [comment]
Please use the mailinglist to ask questions before opening new issues.

You need to setup your private and public address for videobridge:
https://github.com/jitsi/jitsi-videobridge/blob/master/doc/tcp.md#orgjitsivideobridgenat_harvester_local_address
